assets/js/modules/ads/components/common/AdBlockerWarning.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Ads AdBlockerWarning component.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+
+import { MODULES_ADS } from '../../datastore/constants';
+import { CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
+import AdBlockerWarningMessage from '../../../../components/AdBlockerWarningMessage';
+const { useSelect } = Data;
+
+export default function AdBlockerWarning( { context = '' } ) {
```

Comment:
I can't find anywhere this prop is actually used, should we remove it for now?

![Screenshot 2024-04-16 at 21 24 03](https://github.com/google/site-kit-wp/assets/90871/188871b8-8cb5-4e7a-867e-a994da4073f2)